docs/specs/openapi.yaml

```
@@ -153,6 +79,314 @@ paths:
+                      summary: "Perform HTTP check \"svc1-up\""
+                      status: Abort
+                      tasks: 
+                        - id: 76
+                          kind: perform-check
+                          summary: "Perform HTTP check \"svc1-up\""
+                          status: Abort
+                          progress:
+                            label: ""
+                            done: 1
+                            total: 1
+                          spawn-time: "2024-12-22T20:14:54.987825544+08:00"
+                      ready: false,
+                      spawn-time: "2024-12-22T20:14:54.987829502+08:00"
+  /v1/changes/{id}/wait:
+    get:
+      summary: Wait for a change to complete
+      description: |
+        Wait for the change to be finished.
+        
+        If the wait operation succeeds, the result will have the "err" field set to an appropriate error message if the change itself had an error.
+      tags:
+        - changes
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          required: true
+          description: The ID of the change to wait for.
+        - in: query
+          name: timeout
+          schema:
+            type: string
+          description: |
+            Optional timeout (a [duration](#duration)).
+            If specified, wait till the change is ready or a timeout occurs, whichever is first.
+            If not specified or zero, wait indefinitely until the change is ready.
+      responses:
+        '200':
+          description: Change information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetChange"
+              examples:
+                start service:
+                  value:
+                    type: sync
+                    status-code: 200
+                    status: OK
+                    result:
+                      id: "6"
+                      kind: "start"
+                      summary: "Start service \"svc1\""
+                      status: "Done"
+                      tasks:
+                        - id: "4"
+                          kind: "start"
+                          summary: "Start service \"svc1\""
+                          status: "Done"
+                          progress:
+                            label: ""
+                            done: 1
+                            total: 1
+                          spawn-time: "2024-12-12T12:50:29.743512342+08:00"
+                          ready-time: "2024-12-12T12:50:30.751092759+08:00"
+                      ready: true
+                      spawn-time: "2024-12-12T12:50:29.743570175+08:00"
+                      ready-time: "2024-12-12T12:50:30.751094634+08:00"
+  /v1/system-info:
+    get:
+      summary: Get system information
+      description: |
+        Get Pebble daemon's version and boot-id of the system.
```

Minor suggestion for consistency between descriptions:
```suggestion
        Get the version of the Pebble daemon and the boot ID of the system.
```